README.md

```diff
@@ -147,11 +147,10 @@ The following options are available for configuring this serializer:
 
             idstrategy = "incremental"
 
-            # Define a default size for serializer pool
-            # Try to define the size to be at least as big as the max possible
-            # number of threads that may be used for serialization, i.e. max
-            # number of threads allowed for the scheduler
-            serializer-pool-size = 16
+			# Define a default queue builder, by default ConcurrentLinkedQueue is used.
+			# To pass your own queue builder implement the trait KryoSerializer.QueueBuilder
+			# useful for paranoid GC users that want to use JCtools MpmcArrayQueue for example.
+			# custom-queue-builder = "com.github.romix.akka.MyMpmcQueueBuilder"
```

Comment:
I wonder what editor you use, that's IntelliJ Typesafe config editor plugin part of Scala plugin for IntelliJ.